### ISSUE TITLE
Plugin bricked my WP website

### ISSUE THREAD

**Issue by anim8rDev** [justinwhall/wp-buildhook-deploy#1]
So, I installed the plugin and then activated.  My WP website is now bricked.  Here is the error I'm getting trying to view my website...

`Fatal error: Uncaught Error: Class 'LBN_Assets' not found in /wp-content/plugins/littlebot-netlify/littlebot-netlify.php:144 Stack trace: #0 /wp-content/plugins/littlebot-netlify/littlebot-netlify.php(217): Littlebot_Netlify->includes() #1 /wp-includes/class-wp-hook.php(286): Littlebot_Netlify->init('') #2 /wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array) #3 /wp-includes/plugin.php(453): WP_Hook->do_action(Array) #4 /wp-settings.php(450): do_action('init') #5 /wp-config.php(89): require_once('/home/toonstru/...') #6 /wp-load.php(37): require_once('/home/...') #7 /wp-admin/admin.php(31): require_once('/home/...') #8 /hom in /wp-content/plugins/littlebot-netlify/littlebot-netlify.php on line 144`

Any ideas?

**Comment by anim8rDev** [comment]
My problem.  Installed incorrectly.

**Comment by robinmetral** [comment]
@pdxDev I'm having the same issue, how did you fix it?

**Comment by anim8rDev** [comment]
> @pdxDev I'm having the same issue, how did you fix it

I'm sorry, I do not remember exactly.  I *think* that I might of hastily uploaded the directory contained in the zip file instead of the zip file itself.  I could very well be wrong, it has been a while.  (To unbrick my site, I believe that I FTP'd in and deleted the littlebot files/directory and it returned to normal.)

**Comment by robinmetral** [comment]
No worries @pdxDev, thanks for your reply! Unbricking is fine but I haven't found a way to make it work yet :slightly_smiling_face:

**Comment by anim8rDev** [comment]
I'm sorry I cannot help you.  I hadn't used it in a while, so I thought there may be a problem with the newer versions of WP.  However, I just checked and the littlebot plugin is still triggering a Netlify build/deploy when updating a Wordpress page.   

Have you tried installing the zipped plugin through WP ("Add New" plugin) instead of uploading via FTP?    I realize it shouldn't matter, but it may be worth a shot.  I know that is how I ended up installing it.
